js/okex.js

```diff
@@ -1811,9 +1811,12 @@ module.exports = class okex extends Exchange {
             request = this.extend (request, {
                 'type': type, // 1:open long 2:open short 3:close long 4:close short for futures
                 'size': size,
-                'price': this.priceToPrecision (symbol, price),
                 // 'match_price': '0', // Order at best counter party price? (0:no 1:yes). The default is 0. If it is set as 1, the price parameter will be ignored. When posting orders at best bid price, order_type can only be 0 (regular order).
             });
+            // order_type == 4 is market order, cannot fill price
+            if (this.safeString(params, 'order_type', '0') != '4') {
```

Comment:
Can you please elaborate a bit on this edit or post a link to OKEX API docs?

Reply:
Actually, nevermind, found it here: https://www.okex.com/docs/en/#futures-orders. Thank you!